appointment/upload-documents/uploadDocumentsController.js

```diff
@@ -13,16 +13,15 @@ define('uploadDocumentsController', [], function() {
 		$scope.invalidClientInformation = false;
 		$scope.isResidentialAppointment = true;
 
-		// Agree to virtual preparation variables
-		// For some reason, you can't bind the checkbox to a primitive boolean, it has to be in an object: https://stackoverflow.com/a/23943930
-		$scope.agreeToVirtualPreparationCheckbox = {
-			checked: false
+		// Consent variables
+		$scope.consentData = {
+			reviewConsent: 0,
+			virtualConsent: 0,
```

Comment:
I just read online that for PHP to TINYINT in MySQL you should use integers. https://stackoverflow.com/questions/44539117/php-insert-boolean-values-into-mysql

Reply:
Damn, you are sharp! Good catch. Changed those to 0's during testing. Let me know what you think about the 0/1 over true/false.

Reply:
Initializing them as null prob isn't best practice... Can't start them off as zero because that would be valid input for the first value, and I don't want that answer to be initialized for them (even though the form should make them select something anyway). Set it = -1? Or just try a t/f implementation

Reply:
I would use true/false in the FE and then convert it to the 1/0 if necessary on the BE. I think true/false is way easier to work with and more expected--no reason to force the FE to follow some strange convention on the BE

And is there anything wrong with initializing them as null? i.e. that functions, right?

Reply:
I converted them back to booleans on the BE now. Initializing as null does function!